omod/src/main/java/org/openmrs/module/spa/servlet/SpaServlet.java

```diff
@@ -55,10 +64,57 @@ protected long getLastModified(HttpServletRequest servletRequest) {
 
     @Override
     protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
-        if (SpaModuleUtils.isRemoteAssetsEnabled()) {
-            handleRemoteAssets(request, response);
+        String requestURI = request.getRequestURI();
+        if (requestURI != null && requestURI.contains("frontend/config")) {
+            // get the config.json file
+            File configFile = getJsonConfigFile();
+            if (configFile.exists()) {
+                response.setContentType("application/json");
+                response.setHeader("Content-Disposition", "attachment; filename=\"" + configFile.getAbsolutePath() + "\"");
```

Comment:
We do not need to set this header. It does the wrong thing.

Reply:
On the other hand, we probably _do_ want to use many of the other headers set in `handleLocalAssets`